manual/text/font_style.rb

```diff
@@ -20,6 +25,11 @@
     styles.each do |style|
       font example_font, style: style
       text "I'm writing in #{example_font} (#{style})"
+
+      font_style style
+      text "This is also #{style}"
+
+      text "And this is also #{style}", style: style
```

Comment:
I get what you want here but since setting the style using `#font_style` or inline is a no-op, I don't think it's very illustrative. Maybe use the block form of `#font` and `#font_style` to make it clearer?